Makefile

```diff
@@ -59,3 +59,9 @@ html-noplot: clean prep-docs
 
 linkcheck-files:
 	NAPARI_APPLICATION_IPY_INTERACTIVE=0 sphinx-build -b linkcheck -D plot_gallery=0 --color docs/ docs/_build ${FILES} -D sphinx_gallery_conf.examples_dirs=$(GALLERY_PATH) $(SPHINXOPTS)
+
+fallback-videos:
+	for video in $(basename $(wildcard docs/_static/images/*.webm)); do \
+		echo $$video; \
+		ffmpeg -i $$video.webm -vf "scale=trunc(iw/2)*2:trunc(ih/2)*2" -c:v libx264 -crf 18 -preset slow -c:a aac -b:a 192k -strict -2 -y $$video.mp4; \
+	done
```

Comment:
We should call `ffmpeg` only when `mp4` file does not exist or is older than the source video. 
Otherwise each time makefile is called the new binary file will be created (because creation date will be different). 

Here sample code that I found:

```Makefile
VIDEOS_DIR := ./videos
WEBM_FILES := $(wildcard $(VIDEOS_DIR)/*.webm)
MP4_FILES := $(WEBM_FILES:.webm=.mp4)

.PHONY: all
all: $(MP4_FILES)

%.mp4: %.webm
	ffmpeg -i $< -c:v libx264 -preset slow -crf 22 -c:a aac -b:a 128k $@

clean:
	rm -f $(MP4_FILES)
```

It will simplify local testing

Reply:
I did it slightly differently for visibility, but the effect is the same. Thanks!